cvat-core/tests/api/projects.js

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2019-2020 Intel Corporation
+// Copyright (C) 2021 Intel Corporation
```

Comment:
2021 is invalid. It should be 2019-2020 and you do not need to change header in files which aren't touched in the PR.